miner/worker.go

```diff
@@ -751,6 +767,7 @@ func (w *worker) resultLoop() {
 			// Commit block and state to database.
 			_, err := w.chain.WriteBlockWithState(block, receipts, logs, task.state, true)
 			if err != nil {
+				l2ResultTimer.Update(time.Since(startTime))
```

Comment:
can use another timer for the failing path. wdyt? @vyzo @Thegaram

Reply:
I think this case almost never happens, so we don't need another timer for this.